avto-bus/avto-bus.js

```diff
@@ -0,0 +1,35 @@
+function showMinimum(wheels) {
+   if(wheels % 6 === 0) {
+       console.log("Minimum number of busses is:" + wheels/6)
+   }
+}
+
+function showMaximum(wheels) {
+    if(wheels % 4 === 0){
+        console.log("Maximum number of busses is:" + wheels/4)
```

Comment:
raczej uzywaj returnow zamiast console logow. 
Bedzie to wymagalo przerobienia tego kodu bo teraz jak zrobisz same returny to nie bedziesz widzial efektu. Lub to nie zadziala jak powinno. Ale to juz do sprawdzenia i do uruchomienia kodu przez Ciebie

Reply:
Poprawione, patrz dalej.